### PR TITLE
DM: add the description about support dump files store in s3

### PR DESCRIPTION
**Pull request by WizardXiao**
### What is changed, added or deleted? (Required)
Add the description about support dump files store in s3.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.0 (TiDB 6.0 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): [#4361](https://github.com/pingcap/tiflow/pull/4361)

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: dm/task-configuration-file-full.md [hunk 1]
```diff
@@ -103,7 +103,11 @@ mydumpers:                           # dump 处理单元的运行配置参数
 loaders:                             # load 处理单元的运行配置参数
   global:                            # 配置名称
     pool-size: 16                    # load 处理单元并发执行 dump 处理单元的 SQL 文件的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
-    dir: "./dumped_data"             # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。同实例对应的不同任务必须配置不同的目录
+
+    # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。
+    # 支持配置为本地文件系统路径，也支持配置为 Amazon S3 路径，如: s3://dm_bucket/dumped_data?region=us-west-2&endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
+    # 注意：同实例对应的不同任务必须配置不同的目录
```
Comment (Ehco1996) [inline dm/task-configuration-file-full.md:109]:
```suggestion
    # 注意：当使用 S3 时同实例对应的不同任务必须配置不同的目录
```

**Review by Ehco1996 — APPROVED** [review]
rest LGTM

File: dm/task-configuration-file-full.md [hunk 2]
```diff
@@ -103,7 +103,11 @@ mydumpers:                           # dump 处理单元的运行配置参数
 loaders:                             # load 处理单元的运行配置参数
   global:                            # 配置名称
     pool-size: 16                    # load 处理单元并发执行 dump 处理单元的 SQL 文件的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
-    dir: "./dumped_data"             # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。同实例对应的不同任务必须配置不同的目录
+
+    # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。
+    # 支持配置为本地文件系统路径，也支持配置为 Amazon S3 路径，如: s3://dm_bucket/dumped_data?region=us-west-2&endpoint=s3-website.us-east-2.amazonaws.com&access_key=s3accesskey&secret_access_key=s3secretkey&force_path_style=true
+    # 注意：当使用 S3 时同实例对应的不同任务必须配置不同的目录
```
Comment (ran-huang) [inline dm/task-configuration-file-full.md:109]:
```suggestion
    # 注意：当使用 S3 时，同实例对应的不同任务必须配置不同的目录
```

**Review by ran-huang — COMMENTED** [review]
Rest LGTM

File: dm/task-configuration-file-full.md [hunk 3]
```diff
@@ -103,7 +103,11 @@ mydumpers:                           # dump 处理单元的运行配置参数
 loaders:                             # load 处理单元的运行配置参数
   global:                            # 配置名称
     pool-size: 16                    # load 处理单元并发执行 dump 处理单元的 SQL 文件的线程数量，默认值为 16，当有多个实例同时向 TiDB 迁移数据时可根据负载情况适当调小该值
-    dir: "./dumped_data"             # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。同实例对应的不同任务必须配置不同的目录
+
+    # dump 处理单元输出 SQL 文件的目录，同时也是 load 处理单元读取文件的目录。该配置项的默认值为 "./dumped_data"。
```
Comment (sunzhaoyang) [inline dm/task-configuration-file-full.md:107]:
```suggestion
    # 保存上游全量导出数据的目录。该配置项的默认值为 "./dumped_data"。
```

File: dm/task-configuration-file-full.md (hunk 1 above)
Comment (WizardXiao) [reply dm/task-configuration-file-full.md]: This limit will be deleted, because we will add sfffix or subdir from different task and source.

**Review by ran-huang — APPROVED** [review]

**Review by Ehco1996 — APPROVED** [review]

**Review by sunzhaoyang — APPROVED** [review]
lgtm

**Pull request merged**